deps/chakrashim/src/v8string.cc

```diff
@@ -101,12 +101,19 @@ int String::Write(uint16_t *buffer, int start, int length, int options) const {
 int String::WriteOneByte(
     uint8_t* buffer, int start, int length, int options) const {
   size_t count = 0;
-  if (JsCopyString((JsValueRef)this, start, length,
-                    (char*)buffer, &count) == JsNoError) {
+  // JSRT does not have a start parameter on JsCopyString, use
```

Comment:
The comment here is a bit misleading. Even if JsCopyString supports start parameter, we probably cannot use it, as it returns utf8 encoding. If the string contains multibyte chars, JsCopyString result will be incompatible with expected OneByte behavior.